funding/commitment_type_negotiation.go

```diff
@@ -50,6 +50,23 @@ func explicitNegotiateCommitmentType(channelType lnwire.ChannelType,
 	channelFeatures := lnwire.RawFeatureVector(channelType)
 
 	switch {
+	// Lease script enforcement + anchors zero fee + static remote key
+	// features only.
+	case channelFeatures.OnlyContains(
```

Comment:
Can't help but dwell on how much more compact all this could've been if there was a single type here /shruggie